CORE_DEVELOPER.txt

```diff
@@ -0,0 +1,164 @@
+Core Contributor Guide
+======================
+
+Welcome, new core developer!  The core team appreciate the quality of
+your work, and enjoy working with you; we have therefore invited you
+to join us.  Thank you for your numerous contributions to the project
+so far.
+
+This document offers guidelines for your new role.  First and
+foremost, you should familiarize yourself with the project's
+`mission, vision, and values
+<https://github.com/scikit-image/scikit-image/pull/3585>`__.  When in
+doubt, always refer back here.
+
+As a core team member, you gain the responsibility of shepherding
+other contributors' through the review process; here are some
+guidelines.
+
+All Contributors Are Treated Similarly
+--------------------------------------
+
+You now have the ability to push changes directly to the master
+branch, but should never do so; instead, continue making PRs as
+before.
+
+As a core contributor, you gain the ability to merge merge or approve
+other contributors' pull requests.  Much like nuclear launch keys, it
+is a shared power: you must merge *only after* another core has
+approved the PR, *and* after you yourself have carefully reviewed it.
+(See “Reviewing” and especially “If you merge it, you buy it”,
+below.)
+
+Reviewing
+---------
+
+How to Conduct A Good Review
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Always* be kind to contributors. Nearly all of `scikit-image` is
+volunteer work, for which we are tremendously grateful. Provide
+constructive criticism on ideas and implementations, and remind
+yourself of how it feels when your own work is being evaluated.
```

Comment:
Here I might say something like "how it felt when your own work was being evaluated as a novice" or somesuch. How I feel about evaluation now is much more "robust" than when I was first starting out. A negative review from you these days is like "Pffft, here's @stefanv foiling me again! Now I have to explain myself! How annoying!" =P But, in the early days, it might have driven me away altogether. So I want to emphasise this difference.

One might also say "how you like your own contributions to be evaluated" if you want to get at the more professional end of things, where people sometimes might still resort to personal attacks — but not on *our* project! =)